### PR TITLE
🐛 #2178 Proper Analysis Tools Concatenation

### PR DESCRIPTION
**Pull request by demariadaniel**
**Description of changes**

Displays multiple analysis tool entries as a comma + space separated sentence

https://app.zenhub.com/workspaces/icgc-argo-platform-dk-production-board-5e542d38415f5034e9fed89d/issues/icgc-argo/platform-ui/2178

**Type of Change**

- [x] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [x] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- UI Kit (select one):
  - [ ] New Components with storybook stories created
  - [ ] Modified Existing components and updates stories
  - [x] No new UIKit components or changes
- [x] Feature is minimally responsive
- [x] Manual testing of UI feature
- [x] Add copyrights to new files
- [x] Connected ticket to PR

### CODE REVIEW

**Comment by demariadaniel** [comment]
Screen Shot 2022-02-02 at 12 45 59 PM

File: components/pages/file-entity/DataAndAnalysis/index.tsx [hunk 1]
```diff
@@ -98,26 +133,26 @@ const DataAndAnalysis = ({ data }: { data: DataAnalysisInfo }) => {
...
+        `}
+      >
         N/A
       </div>
     ),
     'Workflow Name': getWorkflowNameLink(data.workflowType),
     'Workflow Version': getWorkflowVersionLink(data.workflowType),
-    'Analysis Tools': data.software,
+    'Analysis Tools':
+      data.software && data.software.reduce((prev, current) => `${prev}, ${current}`),
```
Comment (mistryrn) [inline components/pages/file-entity/DataAndAnalysis/index.tsx:155]: Such a smart way to do this! 🤯

**Review by mistryrn — APPROVED** [review]
Looks good! 👍

File: components/pages/file-entity/DataAndAnalysis/index.tsx (hunk 1 above)
Comment (demariadaniel) [reply components/pages/file-entity/DataAndAnalysis/index.tsx:155]: Awww thank you!

**Pull request merged**